interfaces/backends/backends.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 	"github.com/snapcore/snapd/interfaces/apparmor"
 	"github.com/snapcore/snapd/interfaces/dbus"
+	"github.com/snapcore/snapd/interfaces/kernel_module"
```

Comment:
Sorry for entering bike-shed territory right away. I don't like the "kernel_modules" name, it does not feel go-ish. Maybe `kmod`? But probably something for Gustavo as he is really good with names, OTOH we can keep it for now and change it later when he is back.